src/test/java/io/smartcat/migration/CassandraMetadataAnalyzer.java

```diff
@@ -0,0 +1,34 @@
+package io.smartcat.migration;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.common.collect.Iterables.tryFind;
+
+import com.datastax.driver.core.ColumnMetadata;
+import com.datastax.driver.core.KeyspaceMetadata;
+import com.datastax.driver.core.Metadata;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.TableMetadata;
+import com.google.common.base.Optional;
+
+public class CassandraMetadataAnalyzer {
+
+    private Session session;
+
+    public CassandraMetadataAnalyzer(Session aSession) {
```

Comment:
Please use standard naming convention for session parameter and 'this' for private field access.